server/container/cli/src/main/java/org/apache/james/cli/probe/impl/JmxDataProbe.java

```diff
@@ -27,13 +27,12 @@
 import javax.management.MalformedObjectNameException;
 
 import org.apache.james.domainlist.api.DomainListManagementMBean;
-import org.apache.james.probe.DataProbe;
 import org.apache.james.rrt.api.RecipientRewriteTableManagementMBean;
 import org.apache.james.rrt.lib.Mappings;
 import org.apache.james.user.api.UsersRepositoryManagementMBean;
 import org.apache.james.util.MDCBuilder;
 
-public class JmxDataProbe implements DataProbe, JmxProbe {
```

Comment:
Is JmxDataProbe exposed as JMX? (I guess so).

Reply:
No, the MBeans are, the JMX probe is just a way to query the MBean (it's defined on the CLI side thus it does not exist on the server side)

Reply:
Ok.